src/VAC/View.cpp

```diff
@@ -1678,6 +1859,11 @@ QImage View::drawToImage(Time t, double x, double y, double w, double h, int IMG
     return res;
 }
 
+void View::drawLine()
+{
+
```

Comment:
intentionally empty or mistake? Add a comment to explain what's needed here, or just // empty because xxx...

Reply:
Done